modules/bridge/src/main/ts/ephox/bridge/components/content/Autocompleter.ts

```diff
@@ -87,4 +88,4 @@ export const createAutocompleterItem = (spec: AutocompleterItemSpec): Result<Aut
   StructureSchema.asRaw<AutocompleterItem>('Autocompleter.Item', autocompleterItemSchema, spec);
 
 export const createAutocompleter = (spec: AutocompleterSpec): Result<Autocompleter, StructureSchema.SchemaError<any>> =>
-  StructureSchema.asRaw<Autocompleter>('Autocompleter', autocompleterSchema, spec);
+  StructureSchema.asRaw<Autocompleter>('Autocompleter', autocompleterSchema, { ...spec, trigger: Obj.get(spec, 'trigger').getOr(spec.ch) });
```

Comment:
Is there any plans or issues to remove this stopgap fix in the future? It's for backwards compatibility, sure, but we'd probably prefer to remove it at some point, and not forgetting would be nice.

Reply:
I've created TINY-8929 for that

Reply:
Could this be simplified to this?
```suggestion
  StructureSchema.asRaw<Autocompleter>('Autocompleter', autocompleterSchema, { trigger: spec.ch, ...spec });
```